regression/cbmc-cover/mcdc1/test.desc

```diff
@@ -10,6 +10,6 @@ main.c
 ^\[main.coverage.5\] file main.c line 14 function main MC/DC independence condition `C && D && E && !A && B.*: SATISFIED$
 ^\[main.coverage.6\] file main.c line 14 function main MC/DC independence condition `C && D && E && !A && !B.*: SATISFIED$
 ^\*\* .* of .* covered \(100.0%\)$
-^\*\* Used 10 iterations$
+^\*\* Used 
```

Comment:
I believe we can safely remove that line. (And the same holds for all the other "Used" below.)

Reply:
I'll add another commit that drops all the lines that only report "Used". The other lines I cut still contain some useful information.